res/enkf/local_obsdata.py

```diff
@@ -67,16 +58,12 @@ def __getitem__(self, key):
             if key < 0:
                 key += len(self)
             if 0 <= key < len(self):
-                node_ = self._iget_node(key)
-                node_.setParent(self)
-                return node_
+                return _lib.local.local_obsdata.iget_node(self, key)
             else:
                 raise IndexError("Invalid index, valid range is [0, %d)" % len(self))
         else:
```

Comment:
Do you know if this functionality is needed? Or if the i-get is needed? I.e. do we need both i get and "string" get?

Reply:
The purpose of the `iget`method is to provide iteration support (along with the `__len__()`) method - I guess that could be implemented with `__iter__()` instead.

Reply:
Yes, idealy we would do that, I know this PR does not touch that part. Do you know if both methods are used in the code currently?

Reply:
I "know" that the iteration construction:
```Python
for obs in local_obs:
      // do something
```
makes sense - whether the `iget()` is used by itself I do not know.

Reply:
You are not changing the behavior here, so is fine to leave it as is.